src/pages/individual/EditIndividualMetadata.jsx

```diff
@@ -52,32 +60,43 @@ export default function EditIndividualMetadata({
   metadata = metadata || [];
   // hotfix //
 
-  const defaultFieldMetadata = metadata.filter(
-    field => !field.customField,
-  );
-  const customFieldMetadata = metadata.filter(
-    field => field.customField,
-  );
-
   const [defaultFieldValues, setDefaultFieldValues] = useState(
-    getInitialFormValues(defaultFieldMetadata, 'name'),
+    getInitialFormValues(getDefaultFieldMetadata(metadata), 'name'),
   );
 
   const [customFieldValues, setCustomFieldValues] = useState(
-    getInitialFormValues(customFieldMetadata, 'id'),
+    getInitialFormValues(getCustomFieldMetadata(metadata), 'id'),
   );
 
   const [formErrors, setFormErrors] = useState([]);
 
   const showErrorAlert =
     patchIndividualError || formErrors.length > 0;
 
+  const handleClose = useCallback(
+    () => {
+      setDefaultFieldValues(
+        getInitialFormValues(
+          getDefaultFieldMetadata(metadata),
+          'name',
+        ),
+      );
+      setCustomFieldValues(
+        getInitialFormValues(getCustomFieldMetadata(metadata), 'id'),
+      );
+      setPatchIndividualError(null);
+      setFormErrors([]);
+      onClose();
+    },
+    [metadata, setPatchIndividualError, onClose],
```

Comment:
not sure `setPatchIndividualError` needs to be included in the dependency array. since this is coming from useState i'm not sure the function will ever change in a meaningful way. was the linter making you do this though? 

i was reading about functions in dependency arrays here: 
https://reacttraining.com/blog/when-to-use-functions-in-hooks-dependency-array/

Reply:
> When do I use functions in a Hooks Dependency Array?
> The shore answer is...*whenever the linter tells you to*.

```
src/pages/individual/EditIndividualMetadata.jsx
  91:5  warning  React Hook useCallback has a missing dependency: 'setPatchIndividualError'. Either include it or remove the dependency array  react-hooks/exhaustive-deps
```

I do agree that we know that setPatchIndividualError is just the setter function from a useState call so it can be safely omitted